tests/cloud_functions/conftest.py

```
@@ -0,0 +1,81 @@
+
+import pytest
+from faker import Faker
+from firebase_admin import delete_app, get_app
+
+from fitnessllm_dataplatform.utils.logging_utils import logger
+
+fake = Faker()
+
+
+@pytest.fixture(autouse=True)
+def cleanup_firebase():
```

Ruff (0.8.2)

Missing return type annotation for public function `cleanup_firebase`

(ANN201)